db/migrations/49_rename_datetime_columns.sql

```diff
@@ -0,0 +1,876 @@
+-- +migrate Up
+
+
+ALTER TABLE `filters`
+    RENAME COLUMN `start_date` TO `started_at`;
+ALTER TABLE `groups`
+    RENAME COLUMN `date_created` TO `created_at`,
+    RENAME COLUMN `code_end` TO `code_expires_at`,
+    RENAME COLUMN `lock_user_deletion_date` TO `lock_user_deletion_until`;
+ALTER TABLE `groups_attempts`
+    RENAME COLUMN `start_date` TO `started_at`,
+    RENAME COLUMN `validation_date` TO `validated_at`,
+    RENAME COLUMN `finish_date` TO `finished_at`,
+    RENAME COLUMN `last_activity_date` TO `last_activity_at`,
+    RENAME COLUMN `thread_start_date` TO `thread_started_at`,
+    RENAME COLUMN `best_answer_date` TO `best_answer_at`,
+    RENAME COLUMN `last_answer_date` TO `last_answer_at`,
+    RENAME COLUMN `last_hint_date` TO `last_hint_at`,
+    RENAME COLUMN `contest_start_date` TO `contest_started_at`;
+ALTER TABLE `groups_groups`
+    RENAME COLUMN `status_date` TO `status_changed_at`;
```

Comment:
Still not clear when I read the name, isn't it `latest_status_changed_at` ?

Reply:
let's skip the 'latest_' prefix when possible, otherwise we would have a lot of them...

Reply:
Ok, but anyway "status" does not mean anything here. Either it is "type_changed_at" or just "changed_at" considering that it is quite implicit (as the other attributes are not really changed, we can consider it is)

Reply:
fixed (will be `type_changed_at`)